ios/Classes/TiMapView.m

```diff
@@ -832,6 +834,25 @@ -(void)showAnnotations:(id)args
     },NO);
 }
 
+-(void)setClusterAnnotation:(TiMapAnnotationProxy *)annotation forMembers:(NSArray *)members
+{
+    if (!clusterAnnProxyDict) {
+        clusterAnnProxyDict = [[NSMutableDictionary alloc] init];
+    }
+    
+    TiMapAnnotationProxy *annotationProxy = [clusterAnnProxyDict objectForKey:members];
+    if (annotationProxy) {
+        [[self proxy] forgetProxy:annotationProxy];
+    }
+    [clusterAnnProxyDict removeObjectForKey:members];
+    [clusterAnnProxyDict setObject:annotation forKey:members];
+}
+
+-(TiMapAnnotationProxy *)getClusterAnnotationProxyForMembers:(NSArray *)members
```

Comment:
do not use the `getXXXX` prefix in Objective-C, just call it `clusterAnnotationProxyForMembers`.